app/scripts/views/mixins/open-confirmation-email-mixin.js

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+/**
+ * A view mixin that helps confirm* views show the "Open Gmail" button
+ * directly from FxA.
+ */
+define(function (require, exports, module) {
+  'use strict';
+
+  var BaseView = require('views/base');
+  var t = BaseView.t;
+
+  return {
+    events: {
+      'click #open-webmail': '_webmailTabOpened'
+    },
+
+    /**
+     * Get the webmail URL for the given email
+     *
+     * @param {string} email
+     * @returns {string}
+     */
+
+    _isGmail: function (email) {
+      return /@gmail\.com$/.test(email);
```

Comment:
@larissagaulia what are your thoughts on this structure (pseudocode): 

```js
var WEBMAIL_SERVICES = [
   {
    buttonName: 'Open Gmail', // used for button
    regex: /@gmail\.com$/, // used for determining the button
    link: ''https://mail.google.com/mail/u/?authuser='
    webmailType: `gmail` // used for the template to determine which was clicked and metrics
   }
]

// generates link from service
_getWebmailLink(email) {
...
}

// determines if button service is supported for given email
_isSupported(email)
```

^ The above structure makes it really easy to add support for new services. If we wanted to add more services we would not have to add more `_is[Thing]` functions and adjust other stuff.